src/main/java/org/folio/service/processing/ParallelFileChunkingProcessor.java

```diff
@@ -174,20 +175,15 @@ protected Future<Void> processFile(FileDefinition fileDefinition,
     readStreamWrapper.pause();
 
     LOGGER.debug("processFile:: Starting to send event to Kafka... jobProfile: {}, eventType: {}", jobProfile, eventType);
-    KafkaProducer<String, String> producer = KafkaProducer.createShared(vertx,
-      eventType + "_Producer", kafkaConfig.getProducerProps());
+
+    KafkaProducer<String, String> producer = new SimpleKafkaProducerManager(vertx, kafkaConfig).createShared(eventType + "_Producer");
```

Comment:
We should omit adding explicitly `"_Producer"` since it's being added in the `createShared()` method

Reply:
Forgot to remove it, thanks)